tests/common/mod.rs

```diff
@@ -32,5 +32,18 @@ pub async fn connect_to_test_db(database_name: impl ToString) -> Result<MongoDb,
     };
     config.database_name = database_name.to_string();
 
-    Ok(MongoDb::connect(&config).await?)
+    let db = MongoDb::connect(&config).await?;
+    db.clear().await?;
+    Ok(db)
+}
+
+#[allow(unused)]
+pub async fn setup_coll<T: MongoDbCollection + Send + Sync>(db: &MongoDb) -> Result<T, TestDbError> {
```

Comment:
Can we please use the full names when it's not extremely long?

Reply:
Done